File: app/src/main/java/com/vpulse/ftpnext/commons/FTPFileUtils.java
package com.vpulse.ftpnext.commons;

import org.apache.commons.net.ftp.FTPFile;

import java.io.File;

public class FTPFileUtils {

    public static String getFileName(FTPFile iFTPFile) {
        if (iFTPFile != null && iFTPFile.getName().contains(File.separator))
            return iFTPFile.getName().substring(iFTPFile.getName().lastIndexOf(File.separator));
        return iFTPFile.getName();
    }
}

File: app/src/main/java/com/vpulse/ftpnext/commons/FTPFileUtils.kt
package com.vpulse.ftpnext.commons

import org.apache.commons.net.ftp.FTPFile
import java.io.File

object FTPFileUtils {
    fun getFileName(iFTPFile: FTPFile?): String {
        return if (iFTPFile != null && iFTPFile.name.contains(File.separator)) iFTPFile.name.substring(
            iFTPFile.name.lastIndexOf(File.separator)
        ) else iFTPFile!!.name
    }
}